src/actions/index.js

```diff
@@ -11,8 +11,10 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {type: 'INCREMENT'};
```

Comment:
This works, but convention is to use the variables defined at the top of this file, as opposed to strings. So `INCREMENT` instead of `'INCREMENT'`. Notice how we export these variables here in action, and in the reducer how we import them? This allows us to use the variables in the switch statement in the reducer as well.